build_binaries.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+git submodule update --init --recursive
```

Comment:
Build script should not modify the submodules, this can be done only once or manually after updating them.

Reply:
Right. Fixed.